oaknational/Oak-Web-Application
# PUPIL-318 - attempt to replace Flex with OakFlex in about us pages

## Description

**the first out of a long series**

the long term goal: to replace components from `SharedComponents` with `oak-components` and eventually deprecate `SharedComponents` 

attempt to replace Flex with OakFlex in about us pages
- changed where possible
- in _Who we are_, I replaced old `Flex` with `OakGrid` and `OakGridArea`

## Issue(s)

Fixes #PUPIL-318

## How to test

1. Go to https://deploy-preview-2203--oak-web-application.netlify.thenational.academy/about-us/who-we-are
2. Check all tabs
3. Check different screen sizes - the layout and spacing should be the same as on the live site (except the image in the 'Meet our teachers' sections on `/partners` tab which is now 240px high rather than 300px)
